### ISSUE TITLE
"This document is available in plain text, HTML and PDF." is not TRUE

### ISSUE THREAD

**Issue by malcook** [fniessen/refcard-org-mode#7]
the pdf version is missing

**Comment by dcunited001** [comment]
yes it is (i think). by that logic, then [Milewski's Category Theory for Programmers](https://github.com/hmemcpy/milewski-ctfp-pdf) is not available in the Haskell version... but the source is available. Somehow the book will simply build in multiple languages, provided implementations follow the overall format. 

If they just gave you the plain PDF, you may not notice all the work that was done. So they stick you with a PDF containing OCaml examples (or something idk ocaml, i showed up for the haskell). It's not hard.

You may have to build it, but the org document itself is probably more valuable than a PDF, although a nicely formatted refcard helps a ton.

If you need help learning enough org-mode or emacs to build it, definitely reach out somewhere and find someone who will mentor you. "take this; do not go alone"
